src/applications/vaos/utils/selectors.js

```diff
@@ -320,9 +329,8 @@ export const isWelcomeModalDismissed = state =>
     announcement => announcement === 'welcome-to-new-vaos',
   );
 
-export const selectFacilities = state =>
-  selectPatientFacilities(state)?.filter(
-    f => !f.facilityId.startsWith('742'),
-  ) || null;
 export const selectSystemIds = state =>
   selectFacilities(state)?.map(f => f.facilityId) || null;
+
+export const selectIsCernerOnlyPatient = state =>
```

Comment:
Didn't see this used anywhere besides the unit test.  Is it for future use?

Reply:
Yeah, it's used in the other Cerner PR. I broke them into two PRs because I thought that'd be easier to review, but I didn't split out the changes inside this file.